app/models/user.rb

```diff
@@ -136,7 +172,7 @@ def create_ldap_user
       cn:           "A User",
       objectclass:  ["person", "inetOrgPerson"],
       uid:          uid,
-      userPassword: (password.blank? ? "{CRYPT}#{encrypted_password}" : password),
+      userPassword: (Rails.env.test? ? password : "{CRYPT}#{encrypted_password}"),
```

Comment:
This is something I haven't been able to solve on Travis with the `slapd` instance we are running there and its configuration. Using `{CRYPT}#{encrypted_password}` will make `spec/features/auth/login_feature_spec.rb` to fail on a couple of specs (the ones related to logging in correctly).

I am just workarounding it this way since I have been trying to find out what's going on for some time now.

Reply:
ideally this explanation would go into the commit message, maybe in an extra commit?

Reply:
or maybe even a comment inline... not sure. but we at least would have some clue next time we look at it and wonder